test/fixtures.js

```diff
@@ -15,7 +15,8 @@ module.exports = function(engine) {
       options = fn;
       fn = function() {};
     }
-    return bemxjst[engine].compile(fn, options || {});
+
+    return bemxjst[engine]._compile(fn, options || {});
```

Comment:
@zxqfox old `compile()` use `generate()`. But for coverage we need to use `require`.